app/Controllers/Page.php

```diff
@@ -90,6 +92,106 @@ public function photoCredits()
         return $photos;
     }
 
+    /*
+    Create a new Wordpress Query for looping `pcc-story` post types.
+    */
+    public function storiesQuery ()
+    {
+        $meta_query_args = array();
+
+        /* If the clear parameter is set, unset all parameters so they
+        aren't queried. */
+        if (get_query_var( 'clear' )) {
+
+            /* TODO: Improve this implementation. Parameters should
+            be unset on submission and before reaching this point. */
+            remove_query_arg ( 'org' );
+            remove_query_arg( 'clear' );
+        } else if (get_query_var( 'org' )) {
+            /* If filtering by a single organization name. */
+            $meta_query_args = [
+              'key' => 'pcc_story_organization',
+              'value' => get_query_var( 'org' )
+            ];
+        }
+
+        $query = new \WP_Query(
+            [
+                'post_type' => 'pcc-story',
+                'posts_per_page' => -1,
+                'post__in' => $postIds,
```

Comment:
Where does the `$postIds` variable come from? I don't see it being passed as input or being set anywhere. If it's not used it should be removed because it isn't affecting the query as is.